tests failing locally with pandas 0.24.2

pandas 0.24.2

<details>

```
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
__________________________________________ ERROR at setup of test_argsort[date] ___________________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
__________________________________________ ERROR at setup of test_argsort[time] ___________________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
________________________________________ ERROR at setup of test_astype_copy[date] _________________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
________________________________________ ERROR at setup of test_astype_copy[time] _________________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_____________________________________ ERROR at setup of test_asdatetime[<M8[ns]-True] _____________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_______________________________________ ERROR at setup of test_asdatetime[<M8-True] _______________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_________________________________ ERROR at setup of test_asdatetime[datetime64[ns]-True] __________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
___________________________________ ERROR at setup of test_asdatetime[datetime64-True] ____________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
____________________________________ ERROR at setup of test_asdatetime[datetime-True] _____________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
____________________________________ ERROR at setup of test_asdatetime[<M8[us]-False] _____________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
____________________________________ ERROR at setup of test_asdatetime[<M8[ms]-False] _____________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_____________________________________ ERROR at setup of test_asdatetime[<M8[s]-False] _____________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_________________________________ ERROR at setup of test_asdatetime[datetime64[us]-False] _________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_________________________________ ERROR at setup of test_asdatetime[datetime64[ms]-False] _________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_________________________________ ERROR at setup of test_asdatetime[datetime64[s]-False] __________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_________________________________________ ERROR at setup of test_astimedelta[<m8] _________________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_______________________________________ ERROR at setup of test_astimedelta[<m8[s]] ________________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_______________________________________ ERROR at setup of test_astimedelta[<m8[ms]] _______________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_______________________________________ ERROR at setup of test_astimedelta[<m8[us]] _______________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_______________________________________ ERROR at setup of test_astimedelta[<m8[ns]] _______________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
______________________________________ ERROR at setup of test_astimedelta[timedelta] ______________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_____________________________________ ERROR at setup of test_astimedelta[timedelta64] _____________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
___________________________________ ERROR at setup of test_astimedelta[timedelta64[s]] ____________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
___________________________________ ERROR at setup of test_astimedelta[timedelta64[ms]] ___________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
___________________________________ ERROR at setup of test_astimedelta[timedelta64[us]] ___________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
___________________________________ ERROR at setup of test_astimedelta[timedelta64[ns]] ___________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
____________________________________________ ERROR at setup of test_any[date] _____________________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
____________________________________________ ERROR at setup of test_any[time] _____________________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
____________________________________________ ERROR at setup of test_all[date] _____________________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
____________________________________________ ERROR at setup of test_all[time] _____________________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_______________________________________ ERROR at setup of test_min_max_median[date] _______________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_______________________________________ ERROR at setup of test_min_max_median[time] _______________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_____________________________________________ ERROR at setup of test_date_add _____________________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
_____________________________________________ ERROR at setup of test_date_sub _____________________________________________

    @pytest.fixture(autouse=True)
    def register_dtype():
>       import db_dtypes  # noqa

tests/unit/test_dtypes.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
db_dtypes/__init__.py:32: in <module>
    from db_dtypes import core
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from typing import Any, Optional, Sequence
    
    import numpy
    import pandas
    from pandas._libs import NaT
>   from pandas._typing import Scalar
E   ModuleNotFoundError: No module named 'pandas._typing'

db_dtypes/core.py:20: ModuleNotFoundError
------- generated xml file: /Users/swast/src/github.com/googleapis/python-db-dtypes-pandas/unit_3.6_sponge_log.xml --------


================================================= short test summary info =================================================
ERROR tests/unit/test_dtypes.py::test_array_construction[None-date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_array_construction[None-time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_array_construction[_from_sequence-date] - ModuleNotFoundError: No module named 'pa...
ERROR tests/unit/test_dtypes.py::test_array_construction[_from_sequence-time] - ModuleNotFoundError: No module named 'pa...
ERROR tests/unit/test_dtypes.py::test_array_construction[_from_sequence_of_strings-date] - ModuleNotFoundError: No modul...
ERROR tests/unit/test_dtypes.py::test_array_construction[_from_sequence_of_strings-time] - ModuleNotFoundError: No modul...
ERROR tests/unit/test_dtypes.py::test_array_construction_bad_vaue_type[date] - ModuleNotFoundError: No module named 'pan...
ERROR tests/unit/test_dtypes.py::test_array_construction_bad_vaue_type[time] - ModuleNotFoundError: No module named 'pan...
ERROR tests/unit/test_dtypes.py::test_time_series_construction[date] - ModuleNotFoundError: No module named 'pandas._typ...
ERROR tests/unit/test_dtypes.py::test_time_series_construction[time] - ModuleNotFoundError: No module named 'pandas._typ...
ERROR tests/unit/test_dtypes.py::test_timearray_comparisons[left0-==-right0-expected0-date] - ModuleNotFoundError: No mo...
ERROR tests/unit/test_dtypes.py::test_timearray_comparisons[left0-==-right0-expected0-time] - ModuleNotFoundError: No mo...
ERROR tests/unit/test_dtypes.py::test_timearray_comparisons[left1-==-right1-expected1-date] - ModuleNotFoundError: No mo...
ERROR tests/unit/test_dtypes.py::test_timearray_comparisons[left1-==-right1-expected1-time] - ModuleNotFoundError: No mo...
ERROR tests/unit/test_dtypes.py::test_timearray_comparisons[left2-<=-right2-expected2-date] - ModuleNotFoundError: No mo...
ERROR tests/unit/test_dtypes.py::test_timearray_comparisons[left2-<=-right2-expected2-time] - ModuleNotFoundError: No mo...
ERROR tests/unit/test_dtypes.py::test_timearray_comparisons[left3-<=-right3-expected3-date] - ModuleNotFoundError: No mo...
ERROR tests/unit/test_dtypes.py::test_timearray_comparisons[left3-<=-right3-expected3-time] - ModuleNotFoundError: No mo...
ERROR tests/unit/test_dtypes.py::test_timearray_comparisons[left4->=-right4-expected4-date] - ModuleNotFoundError: No mo...
ERROR tests/unit/test_dtypes.py::test_timearray_comparisons[left4->=-right4-expected4-time] - ModuleNotFoundError: No mo...
ERROR tests/unit/test_dtypes.py::test_timearray_comparisons[left5->=-right5-expected5-date] - ModuleNotFoundError: No mo...
ERROR tests/unit/test_dtypes.py::test_timearray_comparisons[left5->=-right5-expected5-time] - ModuleNotFoundError: No mo...
ERROR tests/unit/test_dtypes.py::test___getitem___arrayindex[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test___getitem___arrayindex[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_timearray_slicing[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_timearray_slicing[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_item_assignment[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_item_assignment[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_array_assignment[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_array_assignment[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_repeat[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_repeat[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_copy[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_copy[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_from_ndarray_copy[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_from_ndarray_copy[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test__from_factorized[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test__from_factorized[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_isna[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_isna[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test__validate_scalar_invalid[date] - ModuleNotFoundError: No module named 'pandas._typ...
ERROR tests/unit/test_dtypes.py::test__validate_scalar_invalid[time] - ModuleNotFoundError: No module named 'pandas._typ...
ERROR tests/unit/test_dtypes.py::test_take[False-None-date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_take[False-None-time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_take[True-None-date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_take[True-None-time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_take[True-fill_value2-date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_take[True-fill_value2-time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_take[True-nan-date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_take[True-nan-time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_take[True-42-date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_take[True-42-time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_take_bad_index[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_take_bad_index[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test__concat_same_type_via_concat[date] - ModuleNotFoundError: No module named 'pandas....
ERROR tests/unit/test_dtypes.py::test__concat_same_type_via_concat[time] - ModuleNotFoundError: No module named 'pandas....
ERROR tests/unit/test_dtypes.py::test__concat_same_type_not_same_type[date] - ModuleNotFoundError: No module named 'pand...
ERROR tests/unit/test_dtypes.py::test__concat_same_type_not_same_type[time] - ModuleNotFoundError: No module named 'pand...
ERROR tests/unit/test_dtypes.py::test_dropna[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_dropna[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_fillna[date-1-None-None-expect0] - ModuleNotFoundError: No module named 'pandas._t...
ERROR tests/unit/test_dtypes.py::test_fillna[date-value1-None-None-expect1] - ModuleNotFoundError: No module named 'pand...
ERROR tests/unit/test_dtypes.py::test_fillna[date-None-backfill-None-expect2] - ModuleNotFoundError: No module named 'pa...
ERROR tests/unit/test_dtypes.py::test_fillna[date-None-bfill-None-expect3] - ModuleNotFoundError: No module named 'panda...
ERROR tests/unit/test_dtypes.py::test_fillna[date-None-pad-None-expect4] - ModuleNotFoundError: No module named 'pandas....
ERROR tests/unit/test_dtypes.py::test_fillna[date-None-ffill-None-expect5] - ModuleNotFoundError: No module named 'panda...
ERROR tests/unit/test_dtypes.py::test_fillna[date-None-backfill-1-expect6] - ModuleNotFoundError: No module named 'panda...
ERROR tests/unit/test_dtypes.py::test_fillna[date-None-bfill-1-expect7] - ModuleNotFoundError: No module named 'pandas._...
ERROR tests/unit/test_dtypes.py::test_fillna[date-None-pad-1-expect8] - ModuleNotFoundError: No module named 'pandas._ty...
ERROR tests/unit/test_dtypes.py::test_fillna[date-None-ffill-1-expect9] - ModuleNotFoundError: No module named 'pandas._...
ERROR tests/unit/test_dtypes.py::test_fillna[time-1-None-None-expect0] - ModuleNotFoundError: No module named 'pandas._t...
ERROR tests/unit/test_dtypes.py::test_fillna[time-value1-None-None-expect1] - ModuleNotFoundError: No module named 'pand...
ERROR tests/unit/test_dtypes.py::test_fillna[time-None-backfill-None-expect2] - ModuleNotFoundError: No module named 'pa...
ERROR tests/unit/test_dtypes.py::test_fillna[time-None-bfill-None-expect3] - ModuleNotFoundError: No module named 'panda...
ERROR tests/unit/test_dtypes.py::test_fillna[time-None-pad-None-expect4] - ModuleNotFoundError: No module named 'pandas....
ERROR tests/unit/test_dtypes.py::test_fillna[time-None-ffill-None-expect5] - ModuleNotFoundError: No module named 'panda...
ERROR tests/unit/test_dtypes.py::test_fillna[time-None-backfill-1-expect6] - ModuleNotFoundError: No module named 'panda...
ERROR tests/unit/test_dtypes.py::test_fillna[time-None-bfill-1-expect7] - ModuleNotFoundError: No module named 'pandas._...
ERROR tests/unit/test_dtypes.py::test_fillna[time-None-pad-1-expect8] - ModuleNotFoundError: No module named 'pandas._ty...
ERROR tests/unit/test_dtypes.py::test_fillna[time-None-ffill-1-expect9] - ModuleNotFoundError: No module named 'pandas._...
ERROR tests/unit/test_dtypes.py::test_unique[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_unique[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_argsort[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_argsort[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_astype_copy[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_astype_copy[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_asdatetime[<M8[ns]-True] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_asdatetime[<M8-True] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_asdatetime[datetime64[ns]-True] - ModuleNotFoundError: No module named 'pandas._ty...
ERROR tests/unit/test_dtypes.py::test_asdatetime[datetime64-True] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_asdatetime[datetime-True] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_asdatetime[<M8[us]-False] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_asdatetime[<M8[ms]-False] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_asdatetime[<M8[s]-False] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_asdatetime[datetime64[us]-False] - ModuleNotFoundError: No module named 'pandas._t...
ERROR tests/unit/test_dtypes.py::test_asdatetime[datetime64[ms]-False] - ModuleNotFoundError: No module named 'pandas._t...
ERROR tests/unit/test_dtypes.py::test_asdatetime[datetime64[s]-False] - ModuleNotFoundError: No module named 'pandas._ty...
ERROR tests/unit/test_dtypes.py::test_astimedelta[<m8] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_astimedelta[<m8[s]] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_astimedelta[<m8[ms]] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_astimedelta[<m8[us]] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_astimedelta[<m8[ns]] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_astimedelta[timedelta] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_astimedelta[timedelta64] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_astimedelta[timedelta64[s]] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_astimedelta[timedelta64[ms]] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_astimedelta[timedelta64[us]] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_astimedelta[timedelta64[ns]] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_any[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_any[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_all[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_all[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_min_max_median[date] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_min_max_median[time] - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_date_add - ModuleNotFoundError: No module named 'pandas._typing'
ERROR tests/unit/test_dtypes.py::test_date_sub - ModuleNotFoundError: No module named 'pandas._typing'
116 errors in 12.40s
nox > Command py.test --quiet --junitxml=unit_3.6_sponge_log.xml --cov=db_dtypes --cov=tests/unit --cov-append --cov-config=.coveragerc --cov-report= --cov-fail-under=0 tests/unit failed with exit code 1
nox > Session unit-3.6 failed.
nox > Ran multiple sessions:
nox > * lint: success
nox > * unit-3.6: failed
```

</details>

--- a/db_dtypes/core.py
+++ b/db_dtypes/core.py
@@ -17,7 +17,6 @@ from typing import Any, Optional, Sequence
 import numpy
 import pandas
 from pandas._libs import NaT
-from pandas._typing import Scalar
 import pandas.compat.numpy.function
 import pandas.core.algorithms
 import pandas.core.arrays
@@ -173,7 +172,7 @@ class BaseDatetimeArray(
 
     def min(
         self, *, axis: Optional[int] = None, skipna: bool = True, **kwargs
-    ) -> Scalar:
+    ):
         pandas.compat.numpy.function.validate_min((), kwargs)
         result = pandas.core.nanops.nanmin(
             values=self._ndarray, axis=axis, mask=self.isna(), skipna=skipna
@@ -182,7 +181,7 @@ class BaseDatetimeArray(
 
     def max(
         self, *, axis: Optional[int] = None, skipna: bool = True, **kwargs
-    ) -> Scalar:
+    ):
         pandas.compat.numpy.function.validate_max((), kwargs)
         result = pandas.core.nanops.nanmax(
             values=self._ndarray, axis=axis, mask=self.isna(), skipna=skipna

--- a/db_dtypes/core.py
+++ b/db_dtypes/core.py
@@ -170,18 +170,14 @@ class BaseDatetimeArray(
         result = pandas.core.nanops.nanall(self._ndarray, axis=axis, skipna=skipna)
         return result
 
-    def min(
-        self, *, axis: Optional[int] = None, skipna: bool = True, **kwargs
-    ):
+    def min(self, *, axis: Optional[int] = None, skipna: bool = True, **kwargs):
         pandas.compat.numpy.function.validate_min((), kwargs)
         result = pandas.core.nanops.nanmin(
             values=self._ndarray, axis=axis, mask=self.isna(), skipna=skipna
         )
         return self._box_func(result)
 
-    def max(
-        self, *, axis: Optional[int] = None, skipna: bool = True, **kwargs
-    ):
+    def max(self, *, axis: Optional[int] = None, skipna: bool = True, **kwargs):
         pandas.compat.numpy.function.validate_max((), kwargs)
         result = pandas.core.nanops.nanmax(
             values=self._ndarray, axis=axis, mask=self.isna(), skipna=skipna

--- a/noxfile.py
+++ b/noxfile.py
@@ -44,7 +44,6 @@ nox.options.sessions = [
 ]
 
 # Error if a python version is missing
-nox.options.stop_on_first_error = True
 nox.options.error_on_missing_interpreters = True
 
 

--- a/owlbot.py
+++ b/owlbot.py
@@ -51,21 +51,6 @@ s.replace(
     ["noxfile.py"], "google/cloud", "db_dtypes",
 )
 
-
-def place_before(path, text, *before_text, escape=None):
-    replacement = "\n".join(before_text) + "\n" + text
-    if escape:
-        for c in escape:
-            text = text.replace(c, "\\" + c)
-    s.replace([path], text, replacement)
-
-
-place_before(
-    "noxfile.py",
-    "nox.options.error_on_missing_interpreters = True",
-    "nox.options.stop_on_first_error = True",
-)
-
 # There are no system tests for this package.
 old_sessions = """
     "unit",

--- a/tests/unit/test_dtypes.py
+++ b/tests/unit/test_dtypes.py
@@ -63,6 +63,7 @@ def eq_na(a1, a2):
     nna2 = pd.notna(a2)
     return np.array_equal(nna1, nna2) and np.array_equal(a1[nna1], a2[nna2])
 
+
 @pytest.fixture(autouse=True)
 def register_dtype():
     import db_dtypes  # noqa

--- a/tests/unit/test_dtypes.py
+++ b/tests/unit/test_dtypes.py
@@ -58,6 +58,11 @@ SAMPLE_DT_VALUES = dict(
 for_date_and_time = pytest.mark.parametrize("dtype", ["date", "time"])
 
 
+def eq_na(a1, a2):
+    nna1 = pd.notna(a1)
+    nna2 = pd.notna(a2)
+    return np.array_equal(nna1, nna2) and np.array_equal(a1[nna1], a2[nna2])
+
 @pytest.fixture(autouse=True)
 def register_dtype():
     import db_dtypes  # noqa
@@ -575,8 +580,8 @@ def test_date_add():
     dates = pd.Series(dates)
     times = pd.Series(times)
     expect = dates.astype("datetime64") + times.astype("timedelta64")[:2]
-    assert np.array_equal(dates + times[:2], expect, equal_nan=True)
-    assert np.array_equal(times[:2] + dates, expect, equal_nan=True)
+    assert eq_na(dates + times[:2], expect)
+    assert eq_na(times[:2] + dates, expect)
 
     do = pd.Series([pd.DateOffset(days=i) for i in range(4)])
     expect = dates.astype("object") + do
@@ -609,7 +614,7 @@ def test_date_sub():
     dates = pd.Series(dates)
     dates2 = pd.Series(dates2)
     expect = dates.astype("datetime64") - dates2.astype("datetime64")[:2]
-    assert np.array_equal(dates - dates2[:2], expect, equal_nan=True)
+    assert eq_na(dates - dates2[:2], expect)
 
     do = pd.Series([pd.DateOffset(days=i) for i in range(4)])
     expect = dates.astype("object") - do

